client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-content/enterprise-agency-hosting/index.tsx

```diff
@@ -50,28 +51,72 @@ export default function EnterpriseAgencyHosting( { isReferMode }: { isReferMode:
 							<VIPLogo width={ 57 } height={ 25 } /> | { translate( 'Enterprise WordPress' ) }
 						</div>
 
-						<div className="enterprise-agency-hosting__top-subheading">
-							{ translate( 'Custom pricing' ) }
+						<div>
+							{ isReferMode
+								? translate(
+										"Successfully refer you client to WordPress VIP and you'll earn a one-time 5% commission"
```

Comment:
I think this should be:

```suggestion
										"Successfully refer your client to WordPress VIP and you'll earn a one-time 5% commission"
```

Reply:
Fixed! 👍